packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -131,13 +152,24 @@ export function PanelChrome({
 
   const testid = title ? selectors.components.Panels.Panel.title(title) : 'Panel';
 
+  const collapsibleHeader = (
+    <button type="button" className={styles.clearButtonStyles} onClick={toggleOpen}>
+      <Icon name={isOpen ? 'angle-down' : 'angle-right'} />
+      <h6 title={title} className={styles.title}>
+        {title}
```

Comment:
this doesn't get ellipsed when the title is longer than panel dimensions: 
![Screenshot 2023-07-25 at 11 12 12](https://github.com/grafana/grafana/assets/1170767/16138b9b-7e96-410c-b948-03a155571be9)

Reply:
I've fixed this and it should work now